src/MakePIAPortable/MakePIAPortable.csproj

```diff
@@ -38,22 +26,29 @@
     <PackageReference Include="runtime.osx-x64.Microsoft.NETCore.ILAsm" Version="5.0.0" GeneratePathProperty="true" Condition="$([MSBuild]::IsOSPlatform('osx'))" />
   </ItemGroup>
 
-  <Import Project="..\..\build\Debugger.PIAs.NonPortable.Packages.settings.targets" />
+  <!-- Keep in sync with ..\..\build\Debugger.PIAs.NonPortable.Packages.settings.targets -->
 
   <ItemGroup Label="NuGet Packages">
-    <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.15.0" Version="$(Microsoft_VisualStudio_Debugger_Interop_15_0_Version)" />
-    <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.16.0" Version="$(Microsoft_VisualStudio_Debugger_Interop_16_0_Version)" />
+    <PackageReference Include="Microsoft.VisualStudio.Debugger.Interop.10.0" Version="$(Microsoft_VisualStudio_Debugger_Interop_10_0_Version)" GeneratePathProperty="true" />
```

Comment:
>PackageReference [](http://example.com/codeflow?start=5&length=16)

Do you need to mark all of these as private so that the net standard version of these reference dlls isn't copies to $(OutputPath)?

Reply:
Actually, we should just change OutputPath instead.